src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -448,7 +456,7 @@ protected TimeSpan RemainingOr(TimeSpan duration)
         public TimeSpan RemainingOrDilated(TimeSpan? duration)
         {
             if(!duration.HasValue) return RemainingOrDefault;
-            if(duration.IsInfinite()) throw new ArgumentException("max duration cannot be infinite");
```

Comment:
This only considered `-1`, whereas now we check for all possible negative timespan values.